Intersect.Server.Core/Maps/MapController.cs

```diff
@@ -415,15 +414,15 @@ public void RespawnAllInstances()
         /// </summary>
         /// <param name="instanceId"></param>
         /// <returns>Whether or not we needed to create a new map instance</returns>
-        public bool TryCreateInstance(Guid instanceId, out MapInstance newLayer)
+        public bool TryCreateInstance(Guid instanceId, out MapInstance newLayer, Player creator)
         {
             newLayer = null;
             lock (GetMapLock())
             {
                 if (!mInstances.ContainsKey(instanceId))
                 {
                     Log.Debug($"Creating new instance with ID {instanceId} for map {Name}");
-                    mInstances[instanceId] = new MapInstance(this, instanceId);
+                    mInstances[instanceId] = new MapInstance(this, instanceId, creator);
                     mInstances[instanceId].Initialize();
```

Comment:
Create locally, set, then `Initialize()` on the local

Reply:
Done, curious as to why